<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KasirWindow</class>
 <widget class="QWidget" name="KasirWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kasir</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nama Barang</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditNamaBarang"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Harga</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditHarga"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Jumlah</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditJumlah"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTambah">
     <property name="text">
      <string>Tambah</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Nama Barang</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Harga</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Jumlah</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonTotal">
     <property name="text">
      <string>Hitung Total</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTotal">
     <property name="text">
      <string>Total: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>